src/pylint_odoo/checkers/odoo_addons.py

```diff
@@ -1216,6 +1229,24 @@ def visit_functiondef(self, node):
                 there_is_super = True
                 break
 
+        # Verify if super attributes are prohibited methods to override
+        if there_is_super and (
+            self.linter.config.prohibited_method_override or len(DFTL_PROHIBITED_OVERRIDE_METHODS) != 0
```

Comment:
This is a lint in more recent versions, because:

`if x != 0:`' 

May be simplified as:

`if x:`

```suggestion
            self.linter.config.prohibited_method_override or DFTL_PROHIBITED_OVERRIDE_METHODS
```